py_trees/decorators.py

```diff
@@ -196,15 +196,18 @@ def update(self):
         if the timeout is exceeded.
         """
         current_time = time.time()
```

Comment:
Since we are making these changes here, perhaps we should switch to monotonic time?

Reply:
Oh, aye...we're in python3. This is a hangover from python2 days!